### ISSUE TITLE
Startup Straight To Edge

### ISSUE THREAD

**Issue by 1CM69** [tip2tail/t2tWinFormAppBarLib#1]
Thanks for this lib, it's fantastic, exactly what I was looking for.

I have set my form to load straight away with edge set to Top but I see that there is a fraction of a second when the form is initially shown on the screen prior to being displayed at the edge.

I would like to remove this so all the end user sees is the AppBar load straight to the Top edge.

Thanks

Kirk

**Comment by tip2tail** [comment]
Hi Kirk,

Thanks for your positive comments.

My project also suffers from this however it is not causing my users any problems.  I am not sure how to remove this or even if it can be.  However I am will leave this issue here and if anyone wants to submit a Pull Request with a fix I'd be happy to review and merge it.

Thanks,
t2t

**Comment by 1CM69** [comment]
OK, since posting I found this;

https://stackoverflow.com/questions/70272/single-form-hide-on-startup

seemingly a few different ways to possibly achieve this. I’ll go through them tomorrow and report back. 

Kirk

**Comment by 1CM69** [comment]
Right, I tried all the various ways listed at that URL but none worked at all.

What I have found now is this:
https://www.codeproject.com/Articles/3728/C-does-Shell-Part

this does work perfectly without the initial flash on the screen but to use it you have to include the 'ShellBasic.dll' file, this is not really what I want to do.

It would be ideal if the functionality of the project at this URL was added to t2tWinFormAppBarLib to remove the flash on the screen but include it without the need for the extra .dll file

Kirk
